util/src/main/java/io/kubernetes/client/informer/impl/DefaultSharedIndexInformer.java

```diff
@@ -216,7 +216,7 @@ public boolean hasSynced() {
    *
    * @param deltas deltas
    */
-  private void handleDeltas(Deque<MutablePair<DeltaFIFO.DeltaType, KubernetesObject>> deltas) {
+  public void handleDeltas(Deque<MutablePair<DeltaFIFO.DeltaType, KubernetesObject>> deltas) {
```

Comment:
why did this become public?

Reply:
this is for testing, so that we can directly push events to the queue. otherwise we will have to mock out watch api-endpoint which is far more difficult.

Reply:
Can you add the Guava `@VisibleForTesting` annotation

https://guava.dev/releases/19.0/api/docs/com/google/common/annotations/VisibleForTesting.html